consul_exporter.go

```diff
@@ -416,23 +424,26 @@ func main() {
 	kingpin.Flag("consul.allow_stale", "Allows any Consul server (non-leader) to service a read.").Default("true").BoolVar(&queryOptions.AllowStale)
 	kingpin.Flag("consul.require_consistent", "Forces the read to be fully consistent.").Default("false").BoolVar(&queryOptions.RequireConsistent)
 
-	log.AddFlags(kingpin.CommandLine)
-	kingpin.Version(version.Print("consul_exporter"))
+	promlogConfig := &promlog.Config{}
+	flag.AddFlags(kingpin.CommandLine, promlogConfig)
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
+	logger := promlog.New(promlogConfig)
 
-	log.Infoln("Starting consul_exporter", version.Info())
-	log.Infoln("Build context", version.BuildContext())
+	level.Info(logger).Log("msg", "Starting consul_exporter", "version", version.Info())
+	level.Info(logger).Log("msg", "Build context", version.BuildContext())
```

Comment:
Arguments to `Log()` should always be even (key+value) so:

`level.Info(logger).Log("build_context", version.BuildContext())`